lib/Model/Binary.cpp

```diff
@@ -292,39 +292,45 @@ bool Function::verify(bool Assert) const {
 }
 
 bool Function::verify(VerifyHelper &VH) const {
-  if (Type == FunctionType::Fake)
+  if (Type == FunctionType::Fake or Type == FunctionType::Invalid)
```

Comment:
Make sure you have a dedicated commit for changes to the verify functions justifying the reason.

Reply:
This is actually your commit, I didn't touch it, just rebased on top of it.